src/ckb/gen/pcn.rs

```diff
@@ -1,4 +1,4 @@
-// Generated by Molecule 0.7.5
+// Generated by Molecule 0.8.0
```

Comment:
It seems that you are using a newer version of molecule, which may introduce some unexpected problem. I specifically chose to use molecule 0.7.5 because this is the version that ckb-gen-types are built from. See here for more detailed reasoning on this https://github.com/contrun/ckb-pcn-node/blob/main/src/ckb/schema/README.md . `pcn.mol` itself is not changed, but this file is updated.

Reply:
I changed it to 0.7.5, seems the generated rust source code from 0.7.5 are missing some converters, I fixed them with a temporary fix now: https://github.com/contrun/ckb-pcn-node/pull/15/commits/52897b75d4eb485b92d392cfd606ee57ed70dc8c

Reply:
I rebased and changed the target branch of this PR to `channel-actor`.